pennylane/math/__init__.py

```diff
@@ -61,6 +61,7 @@
     tensordot,
     unwrap,
     where,
+    jax_argnums_to_tape_trainable,
```

Comment:
I think thus far this tuple is sorted alphabetically, maybe good to keep it sorted :)

Reply:
Good catch!